.github/workflows/pr_active.yaml

```diff
@@ -0,0 +1,73 @@
+name: Preview Build
+on:
+  pull_request:
+    branches:
+      - staging
+      - main
+      - dev
+    paths:
+      - "kofta/**"
```

Comment:
can you scope this so it skips translations, otherwise we are going to run out of github build minutes very fast

Reply:
I removed builds that only change the language files.